documentation/docs/06-form-actions.md

```diff
@@ -307,7 +310,7 @@ If you provide your own callbacks, you may need to reproduce part of the default
 
 The behaviour of `applyAction(result)` depends on `result.type`:
 
-- `success`, `invalid` — sets `$page.status` to `result.status` and updates `form` and `$page.form` to `result.data`
+- `success`, `invalid` — sets `$page.status` to `result.status` and updates `form` and `$page.form` to `result.data` (no exceptions in contrast to `update` from `enhance`)
```

Comment:
Can you expand on this a bit more? I assume you're referring to that `applyAction` will update `form` regardless of the page's location, in contrast to `update`

Reply:
I adjusted the defaults explanation in the previous section and enhance the sentence in this - hope this helps

Reply:
Yeah I think that's better 👍